platform/admin/ui/src/main/webapp/main.js

```diff
@@ -156,33 +156,35 @@
             }));
         });
 
-        var alerts = new AlertsModel.InsecureAlerts();
-
-        alerts.fetch({
-            success: function () {
-                app.addInitializer(function () {
-                    Application.App.alertsRegion.show(new AlertsView.View({
-                        model: alerts
-                    }));
-                });
-            }
-        });
+        // setup alert banners
+        app.addInitializer(function () {
+            var alerts = new Backbone.Collection([]);
 
-        //refresh the session if keypress  or mouse click events occur
-        SessionRefresherUtil(60000);
+            var backendAlerts = new AlertsModel.BackendAlerts();
+            backendAlerts.fetch();
+            alerts = backendAlerts;
 
-        //redirect upon session timeout
-        $(document).ajaxError(function (_, jqxhr) {
+            $(document).ajaxError(function (_, jqxhr) {
                 if (jqxhr.status === 401 || jqxhr.status === 403) {
-                    app.addInitializer(function () {
-                        Application.App.alertsRegion.show(new AlertsView.View({'model': AlertsModel.Jolokia({'stacktrace': 'Forbidden'})}));
-                    });
+                    var sessionTimeoutAlert = AlertsModel.Jolokia({'stacktrace': 'Forbidden'});
+                    // do not show any other alerts if session timeout
+                    alerts = new Backbone.Collection([sessionTimeoutAlert]);
                 }
+            });
 
-            }
-        );
+            var AlertsCollectionView = Marionette.CollectionView.extend({
+                itemView: AlertsView.View
+            });
+
+            Application.App.alertsRegion.show(new AlertsCollectionView({
+                collection: alerts
+            }));
+        });
+
+        // refresh the session if keypress  or mouse click events occur
+        SessionRefresherUtil(60000);
```

Comment:
I don't know, but I didn't modify this line. It was already there: https://github.com/codice/ddf/blob/master/platform/admin/ui/src/main/webapp/main.js#L175.